internal/token.go

```diff
@@ -962,7 +962,10 @@ func (z *Tokenizer) readStartTag() TokenType {
 		raw = z.startTagIn("xmp")
 	}
 	if !raw {
-		raw = z.hasTag("data-astro-raw")
+		raw = z.hasAttribute("data-astro-raw")
+	}
```

Comment:
Still supports `data-astro-raw` for now!

Reply:
Should there be a deprecation warning?

Reply:
Probably but would be more helpful to handle that part in `core`